### ISSUE TITLE
Error compilation on kernel 6.6.8

### ISSUE THREAD

**Issue by sateuwdie** [lcgamboa/tty0tty#19]
Error on Slackware 15.0 using kernel 6.6.8.

```
make -C /lib/modules/6.6.8/build M=/tmp/MG/tty0tty-git_0124_6.6.8/module modules
make[1]: ingresso nella directory «/usr/src/linux-6.6.8»
  CC [M]  /tmp/MG/tty0tty-git_0124_6.6.8/module/tty0tty.o
/tmp/MG/tty0tty-git_0124_6.6.8/module/tty0tty.c:855:18: error: initialization of ‘ssize_t (*)(struct tty_struct *, const u8 *, size_t)’ {aka ‘long int (*)(struct tty_struct *, const unsigned char *, long unsigned int)’} from incompatible pointer type ‘int (*)(struct tty_struct *, const unsigned char *, int)’ [-Werror=incompatible-pointer-types]
  855 |         .write = tty0tty_write,
      |                  ^~~~~~~~~~~~~
/tmp/MG/tty0tty-git_0124_6.6.8/module/tty0tty.c:855:18: note: (near initialization for ‘serial_ops.write’)
cc1: some warnings being treated as errors
make[3]: *** [scripts/Makefile.build:243: /tmp/MG/tty0tty-git_0124_6.6.8/module/tty0tty.o] Errore 1
make[2]: *** [/usr/src/linux-6.6.8/Makefile:1913: /tmp/MG/tty0tty-git_0124_6.6.8/module] Errore 2
make[1]: *** [Makefile:234: __sub-make] Errore 2
make[1]: uscita dalla directory «/usr/src/linux-6.6.8»
make: *** [Makefile:30: all] Errore 2
```

**Comment by lcgamboa** [comment]
This problem has already been reported in the issue https://github.com/lcgamboa/tty0tty/issues/17#issuecomment-1831948698

**Comment by lcgamboa** [comment]
Fixed in 355b051.
